components/src/dynamo/vllm/omni/args.py

```
@@ -339,6 +343,7 @@ class OmniConfig(DynamoRuntimeConfig):
 
     model: str
     served_model_name: Optional[str] = None
+    served_model_aliases: list[str] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)